DNN Platform/DotNetNuke.Web.Mvc/Helpers/DnnUrlHelper.cs

```diff
@@ -22,6 +22,22 @@ public DnnUrlHelper(ViewContext viewContext)
         {
         }
 
+        public DnnUrlHelper(RequestContext requestContext, IDnnController controller)
+        {
+            Requires.NotNull("requestContext", requestContext);
+            Requires.NotNull("controller", controller);
+
+            UrlHelper = new UrlHelper(requestContext);
+            _controller = controller;
+
+            if (_controller == null)
```

Comment:
I think we don't need this check in this Constructor. I can't think of a case in which `controller` can be null.

Reply:
I agree, it was more or less copied and pasted from the other constructor. I'll update and push new changes